### ISSUE TITLE
Released checkpoint appears unable to satisfy the fully-masked inference contract

### ISSUE THREAD

**Issue by lennardliekfeldt-jpg** [Duoluoluos/LaDA-Band#2]
Released checkpoint appears unable to satisfy the published fully-masked inference contract

We verified the released lada_band_lm1B_total3B.ckpt and all auxiliary assets against their official SHA-256 hashes and reproduced the public inference path independently.

The checkpoint itself loads strictly into the released 1B architecture and shows strong partial-masking capability: on real accompaniment tokens, Top-1 accuracy is 26.3% at 15% masking, 31.6% at 30%, and 39.7% at 50%.

However, at the fully-masked state used to start the published diffusion inference, Top-1 drops to 0.0%, Top-10 to 2.4%, and mean probability of the true token to ~0.0004. Generated sequences consequently collapse into a token distribution unlike valid accompaniment sequences and decode into non-musical/buzzer-like audio.

MuCodec decoding was independently ruled out: ground-truth accompaniment tokens passed through the same Flow/VAE decoding path reconstruct recognizable music, and our first Flow window is bit-identical to the official code2sound implementation.

The released checkpoint metadata is also notable:

global_step = 4950
lr_scheduler.last_epoch = 4950
base_lr = 1e-5
current_lr = 9.9e-6
original checkpoint path: llada_moe_step_step=4950_1B.ckpt

The public exp1_config.yaml references this same checkpoint as resume_ckpt_fp with max_steps: 10000.

Could you confirm whether lada_band_lm1B_total3B.ckpt is the exact final checkpoint used for the paper/demo samples? If not, could the final inference checkpoint or the exact training/inference revision used for those samples be released?
